content/zh-cn/docs/concepts/architecture/mixed-version-proxy.md

```diff
@@ -0,0 +1,189 @@
+---
+reviewers:
+- jpbetz
```

Comment:
Please remove the reviewers block from localized version.
The original may get notified when the localized version is revised,
however, most of the time, they simply just don't speak Chinese.